src/util/specific.py

```diff
@@ -42,6 +43,17 @@ def dispatch_by_dbuuid(cr, version, callbacks):
         func(cr, version)
 
 
+def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
+    rename_module(cr, old_module_name, new_module_name)
+
+    add_to_migration_reports(
+        category="Rename Module",
```

Comment:
To be in line with the other categories for custom tables/columns
```suggestion
        category="Custom modules",
```